### PR TITLE
Updates to cleaner imports to es6

### PR DESCRIPTION
**Pull request by dalefrancis88**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ module.exports = {
     "eqeqeq": [ "error", "always" ],
     "func-call-spacing": [ "error", "never" ],
     "indent": [ "error", 2 ],
-    "linebreak-style": [ "error", "unix" ],
+    "linebreak-style": 0,
```
Comment (evilsoft) [inline .eslintrc.js]: Would it be possible to change this back to error. I would hate for those nasty linebreaks to sneak into the codebase.

File: src/combinators/index.spec.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+const applyTo = require('./applyTo')
+const composeB = require('./composeB')
+const constant = require('./constant')
+const flip = require('./flip')
+const identity = require('./identity')
+const substitution = require('./substitution')
+
+test('combinators entry', t => {
+  t.equal(applyTo, applyTo, 'provides the T combinator (applyTo)')
```
Comment (evilsoft) [inline src/combinators/index.spec.js]: The should test the file exports against the index, so something like:
```js
const index = require('.')

test('combinators entry', t => {
  t.equal(index.applyTo, applyTo, 'provides the T combinator (applyTo)')
  ...
})
```
That way we are testing the availability of the functions on the object returned from a `require('crocks/combinators')`.

Should match this pattern in all the new specs.

File: .eslintrc.js (hunk 1 above)
Comment (dalefrancis88) [reply .eslintrc.js]: This was an accident, wasn't meant to get checked in

**Comment by evilsoft** [comment]
LGTM! Have a Squirrel:
image

**Pull request merged**